pages/common/id.md

```diff
@@ -2,10 +2,18 @@
 
 > Display current user and group identity.
 
+- Display current user id (UID), group id (GID) and groups of which you are a member:
```

Comment:
Display the current user's id (UID), group id (GID) and groups to which they belong.